opencog/cython/opencog/spatial.pyx

```diff
@@ -105,10 +105,9 @@ cdef class EntityRecorder:
     def __cinit__(self):
         pass
 
-    def __init__(self, long addr, AtomSpace atomspace):
+    def __init__(self, long addr):
         self.c_entity_recorder = <cEntityRecorder*> PyLong_AsVoidPtr(addr)
-        self.atomspace = atomspace
-
+ 
```

Comment:
p.s. I'm not an expert at his stuff, someone  else should review and commit as needed. This just caught my eye.